Vision/Vision.bs

```diff
@@ -140,7 +140,8 @@ Markup Shorthands: markdown yes
 		organizational sizes,
 		and more.
 		In order to ensure W3C serves the needs of the entire Web user base, 
-		we also strive to broaden diversity and inclusion for our own participants.
+		we strive to actively include their participation
```

Comment:
This line feels pretty clunky still.

```suggestion
		we work to support people participating to bring these various perspectives,
```

Reply:
I think "support" is not the verb we want here, because that doesn't capture a variety of the ways we include participants. For example, allowing open commenting on our drafts through GH issues and mailing lists, and processing those comments as rigorously as internally sourced comments, is one of the key ways that we are actively including the participation of a wide variety of people; but it's not really something I'd phrase as "supporting".

(Also I'm not sure your phrasing is actually less clunky... Mine might be equally clunky, but at least it's shorter. ;)

Reply:
I don't think this phrasing is more understandable at all, and I had to re-read it several times to be able to parse it vs the original. I would leave it as is.